src/services/expense.service.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const { getNewId } = require('../utils/getNewId.js');
+
+const { Expense } = require('../models/Expense.js');
+
+const getAll = () => Expense.findAll();
```

Comment:
If we make a request to the server, the function must be asynchronous with the `try...catch` handler